Source/Examples/WPF.SharpDX/SimpleDemo/MainViewModel.cs

```diff
@@ -40,15 +40,14 @@ public class MainViewModel : BaseViewModel
         public Color4 DirectionalLightColor { get; private set; }
         public Color4 AmbientLightColor { get; private set; }
 
-
         public MainViewModel()
         {
             // titles
-            this.Title = "Simple Demo";

```

Comment:
Whenver I touch a file that has this many redundant uses of `this`, I remove them.